mtp_api/templates/admin/performance/digitaltakeup/report.html

```diff
@@ -0,0 +1,225 @@
+{% extends 'core/report.html' %}
+{% load i18n %}
+{% load admin_urls %}
+{% load credit %}
+{% load simple_admin %}
+
+{% block extrastyle %}
+  {{ block.super }}
+
+  <style>
+    #takeup-chart {
+      width: 800px;
+      height: 400px;
+      margin: 10px 8px;
+    }
+  </style>
+{% endblock %}
+
+{% block extrahead %}
+  {{ block.super }}
+
+  <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+  <script>
+    google.charts.load('current', {packages: ['corechart']});
+    google.charts.setOnLoadCallback(function () {
+      var chartData = new google.visualization.DataTable({
+        cols: [
+          {type: 'string', label: '{% trans 'Periods' %}'},
+          {type: 'number', label: '{% trans 'NOMIS digital credits' %}'},
+          {type: 'number', label: '{% trans 'NOMIS postal credits' %}'},
+          {type: 'number', label: '{% trans 'Digital credits' %}'},
+          {type: 'boolean', role: 'certainty'},
+          {type: 'number', label: '{% trans 'Scaled postal credits' %}'},
+          {type: 'boolean', role: 'certainty'}
+        ],
+        rows: {{ chart_rows_json|safe }}
```

Comment:
As a general rule I'm not a fan of templating javascript, as it makes it difficult to extract the javascript out to a separate file. I'm not suggesting changing it now, but it might be worth looking at refactoring at some point.

Reply:
fair point :)